backstop/tests/menu.html

```diff
@@ -1759,35 +1759,37 @@ <h3>Borderless button</h3>
       </ul>
     </section>
     <script>
-      const sectionPadding = parseInt(getComputedStyle(document.querySelector('#demo-header')).paddingLeft);
+      window.onload = () => {
```

Comment:
This fixes flakiness with positioning calculations? I was trying to do something in #576 but couldn't figure it out

Reply:
Check new images. This fixes the positioning of the menu.